AndyDorokhin/HW11/coderunner_task_2.py

```diff
@@ -0,0 +1,35 @@
+""" 
+    Write your code bellow to create custom Error named InputError and function check(text).
+    The error must save description of error in data attribute.
+    
+    In data of error must be written:
+    "Short text error" if length of string less than 3,
+    "Long text error" if length of string more than 15,
+    "Type text error" if we try to check not string.
+"""
+class InputError(Exception):
```

Comment:
good job
look at this version of code

class InputError(Exception):
    def __init__(self, data):
        self.data = data

    def __str__(self):
        return repr(self.data)


def check(text):
    if not (isinstance(text, str) and 3 <= len(text) <= 15):
        raise InputError("Input error")
    return True


def test_input(text):
    try:
        print(check(text))
    except InputError as e:
        print(e.data)